src/containers/WarningConfirmThresholdScenario/index.jsx

```diff
@@ -157,8 +157,13 @@ class WarningConfirmThresholdScenario extends React.Component {
 	async unlock() {
 		const { password } = this.state;
```

Comment:
you can get `isWifChangingOnly` from this distructurized  object and probably no need to reverse it value, just swiche parts at if condition statment